app/assets/javascripts/table-sort.es6

```diff
@@ -23,10 +38,19 @@ $(document).ready(function() {
             // parentNode is the element we want to move
             return this.parentNode;
           });
+          
+          //this adds a message to the message div, stating what it is sorted by
+          $("#screen-reader-messages").html("Sorted by " + th.find(".aria-sort-description").text() + endText[sortDirection][th.attr("data-sort-type")]);
```

Comment:
I think there should be a space between the text coming in from the `aria-sort-description` and the end text.

Reply:
I have added a space to the strings in end text - do you think it should be separate?

Reply:
Made separate.